manifest/marshal.go

```diff
@@ -21,3 +21,17 @@ func marshalRequest(obj *types.ManifestRequest) ([]byte, error) {
 	}
 	return buf.Bytes(), nil
 }
+
+func unmarshal(r io.Reader) (*types.Manifest, error) {
+	obj := &types.Manifest{}
+	return obj, jsonpb.Unmarshal(r, obj)
+}
+
+func marshal(obj *types.Manifest) ([]byte, error) {
```

Comment:
obj can be github.com/ovrclk/akash/vendor/github.com/gogo/protobuf/proto.Message